monero/wallet_client.go

```diff
@@ -175,11 +174,11 @@ func (c *walletClient) GetBalance(idx uint64) (*wallet.GetBalanceResponse, error
 	})
 }
 
-// WaitForTransReceipt waits for the passed monero transaction ID to receive
-// numConfirmations and returns the transfer information. While this function will always
-// wait for the transaction to leave the mem-pool even if zero confirmations are
-// requested, it is the caller's responsibility to request enough confirmations that the
-// returned transfer information will not be invalidated by a block reorg.
+// WaitForReceipt waits for the passed monero transaction ID to receive numConfirmations
+// and returns the transfer information. While this function will always wait for the
+// transaction to leave the mem-pool even if zero confirmations are requested, it is the
+// caller's responsibility to request enough confirmations that the returned transfer
+// information will not be invalidated by a block reorg.
```

Comment:
Only the first word (the function name) changed, the rest was auto-formatted to the new width.